### ISSUE TITLE
Unknown event types are shown as "Deleted messages"

### ISSUE THREAD

**Issue by notramo** [syphon-org/syphon#431]
Unknown event types (e.g. call events) are shown as "Deleted message". They are indistinguishable from the wrong placeholder the undecryptable messages #430. Maybe the " Deleted message" placeholder is generated as some fallback solution?
Call start, answer, and hangup events should be displayed in the timeline, and if the user clicks them, a dialog should tell that calling is not yet supported in Syphon, and is under development. First, as a quick solution, it should use a design similar to the previous Element timeline design (not the current, tile version).

**Comment by ereio** [comment]
They're technically encrypted timeline events that are indistinguishable pre-decryption from messages. Going to show default text - like the screenshot - for now.

Once we've fixed all the issues we've discovered this week, I'll start working on a state event timeline within the chat.

<img width="401" alt="Screen Shot 2021-12-19 at 9 34 51 PM" src="https://user-images.githubusercontent.com/14258175/146703693-01c885af-111b-4634-8d4c-9f5f0f782523.png">

I've filtered other call type events away, but if a new event pops up without a `body` value in an encrypted chat, it will default to the "Encrypted Message" text for now.

Lets make another issue to track adding feature functionality around unsupported events since there's quite a bit to unpack surrounding them
